<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>okno2</class>
 <widget class="QDialog" name="okno2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 64, 64);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(123, 26, 255, 255), stop:1 rgba(187, 28, 226, 255));</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,0);
font: 600 28pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Приятного </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,0);
font: 600 28pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Просмотра</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-21</x>
     <y>70</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 85px;

background-color: qlineargradient(spread:pad, x1:0, y1:0.477, x2:1, y2:0.517045, stop:0 rgba(154, 27, 241, 255), stop:1 rgba(168, 27, 235, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 65px;
background-color: rgb(64, 64, 64);</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>84</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="kartinki.qrc">
      <normaloff>:/new/prefix1/kartiki/cinema.png</normaloff>:/new/prefix1/kartiki/cinema.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>41</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: qlineargradient(spread:pad, x1:0.458, y1:0, x2:0.48, y2:1, stop:0 rgba(184, 28, 227, 255), stop:1 rgba(121, 239, 255, 255));</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startMovieButton">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>430</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(239, 239, 239);</string>
   </property>
   <property name="text">
    <string>Начать просмотр</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endMovieButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(239, 239, 239);</string>
   </property>
   <property name="text">
    <string>Закончить просмотр </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 236, 236);
color: rgb(43, 43, 43);
border-radius: 10px;</string>
   </property>
   <item>
    <property name="text">
     <string>Выбирите категорию</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0,0);
font: 600 13pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Что будем смотреть сегодня?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 236, 236);
color: rgb(43, 43, 43);
border-radius: 10px;</string>
   </property>
   <item>
    <property name="text">
     <string>Выбирите жанр</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 236, 236);
color: rgb(43, 43, 43);
border-radius: 10px;</string>
   </property>
   <item>
    <property name="text">
     <string>Выберите для просмотра</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(239, 239, 239);</string>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_5</zorder>
  <zorder>startMovieButton</zorder>
  <zorder>endMovieButton</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>pushButton_5</zorder>
 </widget>
 <resources>
  <include location="kartinki.qrc"/>
 </resources>
 <connections/>
</ui>
